docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -346,6 +349,11 @@ Elasticsearch Trino         Supports
 Pass-through queries
 --------------------
 
+.. note::
+
+    This feature is deprecated and disabled by default. To enable it please use
```

Comment:
Please recommend that PTF (table function based) query pass through should be used instead.